### ISSUE TITLE
Reset function: functionality help

### ISSUE THREAD

**Issue by bigmoby** [wildstray/meian-client#9]
Hi @wildstray ,
please could you give me a clarification about Reset function (
https://github.com/wildstray/meian-client/blob/4842c391de71b4a923d81afa95226d4374a6d5ff/meian.py#L387 ) ?
Specifically, I need to know if passing True performs a Reset of the alarm system or if it executes a Reboot. I would strongly like to avoid bricking my system :-) ...I wanna only Reboot the alarm!

My doubt arose after I saw that the Java function is described like this: 

```
public static byte[] reqRebootHost() {
        try {
            Document document = getDocument();
            setXmlNodeValue(document, "/Root/Host/Reset/Ret", "BOL|T");
            setXmlNodeEmpty(document, "/Root/Host/Reset/Err");
            return getXml(new DOMSource(document.getDocumentElement())).getBytes("UTF8");
        } catch (UnsupportedEncodingException e) {
            e.printStackTrace();
            return null;
        }
    }
```

In other words Reboot request with True command parameter. Instead, in the library you wrote, it seems to be the opposite.

Many thanks in advance,
Fabio
